tests/core/test_tracker_stores.py

```diff
@@ -92,6 +92,25 @@ def test_find_tracker_store(default_domain):
     )
 
 
+def test_find_redis_tracker_store_with_ssl(default_domain):
```

Comment:
I don't think this test is testing anything which wasn't cover before, so I we could just skip this test in my opinion.